ci/Dockerfile

```diff
@@ -117,7 +117,7 @@ RUN mkdir -p /go/bin
 COPY --from=builder /usr/local/bin/* /usr/local/bin/
 
 # copy qmstr-qmstr_spdx_analyzer from its stagr
-COPY --from=qmstr_spdx_analyzer /qmstr-spdx-analyzer/spdx-analyzer /usr/local/bin/qmstr_spdx_analyzer
```

Comment:
Now we have one analyzer called spdx-analyzer (lies in the qmstr repo and searches for spdx identifiers) and this one called qmstr-spdx-analyzer.
I don't like the name cause they are both qmstr analyzers.
Maybe we should name it spdx-tag-analyzer or tag-spdx-analyzer?